src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -85,6 +91,21 @@ internal ImageLoadingTransformer(IHostEnvironment env, string imageFolder = null
             : base(Contracts.CheckRef(env, nameof(env)).Register(nameof(ImageLoadingTransformer)), columns)
         {
             ImageFolder = imageFolder;
+            UseImageType = true;
```

Comment:
Instead of this you can do `this(env, imageFolder, true, columns)`.

Reply:
I get an error when I try that.

Reply:
```
        internal ImageLoadingTransformer(IHostEnvironment env, string imageFolder = null, params (string outputColumnName, string inputColumnName)[] columns)
            : this(env, imageFolder, true, columns)
        {
        }
```

---
In reply to: [330708136](https://github.com/dotnet/machinelearning/pull/4242#discussion_r330708136) [](ancestors = 330708136)